core/shared/src/test/scala/scodec/bits/ByteVectorTest.scala

```diff
@@ -700,4 +700,53 @@ class ByteVectorTest extends BitsSuite {
     assert(ByteVector(2) > ByteVector(1, 2))
     assert(ByteVector(100) < ByteVector(200))
   }
+
+  test("toInputStream") {
+    {
+      val is = ByteVector(0, 1, 2).toInputStream
+      assert(is.available() == 3)
+      assert(Source.fromInputStream(is).map(_.toInt.toString).mkString == "012")
+    }
+    {
+      val is = ByteVector((0 to 9).toArray.map(_.toByte)).toInputStream
+      assert(is.available() == 10)
+      is.readNBytes(5)
```

Comment:
`readNBytes` is available only since Java 11

Reply:
Oops.